### DISCUSSION TITLE
Adding the ability for contributors of a repository to delete discussions their own discussions

### DISCUSSION THREAD

**Discussion by unrealapex** [community/community#3880] [Discussions]
It would be nice to have the ability to delete a discussion if the answer is already clear or the discussion was made by mistake.

**Comment by rehhouari** [comment]
It is already an option, look at the options on the right side:
![Screenshot_20210502_003429](https://user-images.githubusercontent.com/78239768/116797498-4a3e7180-aade-11eb-8b09-982ece4b908c.png)

**Reply by unrealapex** [reply]
Actually, I don't have that option. The only option I have it the ablility to transfer the discussion to another repo GitHub created. You might be apart of a GitHub beta giving you that option. Is the option available to project maintainers only?

**Reply by rehhouari** [reply]
Oh yes I think it is! I maintain the project I took that screenshot from.
Rather unexpected that it doesn't let you delete yours regardless of the project

**Reply by unrealapex** [reply]
I do find that a bit strange, I think adding the ability for non project maintainers to delete their own discussions would better suit this.
